server/controllers/comment.js

```diff
@@ -81,6 +81,38 @@ class Comment {
     const updatedComment = await commentRepo.updateComment(newBody, id);
     return goodHttpResponse(response, 200, 'Comment updated', updatedComment);
   }
+
+  /** Create a new comment
+   * @param {object} request Request Object
+   * @param {object} response Response Object
+   * @returns {object} Comment Object
+   */
+  static async getCommentWithHistory(request, response) {
+    if (!request.isAuthorized) {
+      return badHttpResponse(
+        response,
+        401,
+        'You are not permitted to complete this action',
+      );
+    }
+
+    const { slug, id } = request.params;
+    const article = await articleRepo.getArticleBySlug(slug);
```

Comment:
When an article is deleted is there a cascade that deletes all the comments that belong to the article?

Reply:
No. I will update that now.